### PR TITLE
T3s1 display on page

### PR DESCRIPTION
**Pull request by nesclark**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+function createDbConnection(): PDO
```
Comment (Steph-ski) [inline functions.php:7]: Add in dock block for function

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{
+    $db = new PDO('mysql:host=db; dbname=nhhouse_plants', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
```
Comment (Steph-ski) [inline functions.php:13]: Add in dock block for function

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+function getAllData(PDO $db): array
+{
+    $stmnt = $db->prepare("SELECT * FROM `plants`");
+    $stmnt->execute();
+    return $stmnt->fetchAll();
+
+}
+
+function callPlant(array $plants): string
```
Comment (Steph-ski) [inline functions.php:32]: Add in dock block for function

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.title {
+    padding-top: 35px;
+}
+h1 {
+    font-style: normal;
```
Comment (Steph-ski) [inline styles.css]: not sure this styling is needed, doesn't make a different when unselected in Inspector

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    font-weight: bold;
```
Comment (Steph-ski) [inline styles.css]: not sure this styling is needed, doesn't make a different when unselected in Inspector

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+}
+.collection-image {
+    width:250px;
+    height:250px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+footer {
+    background-color: #FEC868;
+    text-align: center;
```
Comment (Steph-ski) [inline styles.css]: not sure this styling is needed, doesn't make a different when unselected in Inspector

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
```
Comment (Steph-ski) [inline styles.css]: not sure this styling is needed, doesn't make a different when unselected in Inspector

**Review by Steph-ski — COMMENTED** [review]
Webpage looks really good, like the styling. 

HTML put through validator - all fine.

CSS - some styling looks like it can be removed. 

Functions - dock blocks to be added.

Unit tests all passed.

File: functions.php (hunk 1 above)
Comment (nesclark) [reply functions.php:7]: all done

File: functions.php (hunk 2 above)
Comment (nesclark) [reply functions.php:13]: all done

File: functions.php (hunk 3 above)
Comment (nesclark) [reply functions.php:32]: all done

File: styles.css (hunk 4 above)
Comment (nesclark) [reply styles.css]: removed

File: styles.css (hunk 5 above)
Comment (nesclark) [reply styles.css]: removed

File: styles.css (hunk 6 above)
Comment (nesclark) [reply styles.css]: removed

File: styles.css (hunk 7 above)
Comment (nesclark) [reply styles.css]: removed

**Review by Steph-ski — APPROVED** [review]

**Pull request merged**